test/bigtest/tests/units/settings/ChooseJobProfilePage/ChooseJobProfilePage-test.js

```diff
@@ -0,0 +1,190 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { cleanup } from '@bigtest/react';
+import {
+  describe,
+  it,
+  beforeEach,
+  before,
+} from '@bigtest/mocha';
+
+import {
+  mountWithContext,
+  buildResources,
+  buildMutator,
+} from '@folio/stripes-data-transfer-components/interactors';
+import stripesComponentsTranslations from '@folio/stripes-components/translations/stripes-components/en';
+import commonTranslations from '@folio/stripes-data-transfer-components/translations/stripes-data-transfer-components/en';
+
+import { ChooseJobProfileComponent as ChooseJobProfile } from '../../../../../../src/components/ChooseJobProfile';
+import { ChooseJobProfileInteractor } from './interactor';
+import translations from '../../../../../../translations/ui-data-export/en';
+
+const chooseJobProfile = new ChooseJobProfileInteractor();
+
+const translationsProperties = [
```

Comment:
I think that it will be better to move `translationProperties` to helpers, because it used many times in this module's tests

Reply:
Good idea, I also thought about that but forgot to implement, I think now it is the right time of moving it.